### PR TITLE
Set up database

### PR DESCRIPTION
**Pull request by kenash-dev**
Add dbup project to the solution.
create appsetting.json for the project.
add a test script to create a table in database.

### CODE REVIEW

File: src/MovieSeller.sln [hunk 1]
```diff
@@ -17,10 +19,17 @@ Global
 		{EA213EAE-280D-40CD-BF2D-F0123DA1C94C}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{EA213EAE-280D-40CD-BF2D-F0123DA1C94C}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{EA213EAE-280D-40CD-BF2D-F0123DA1C94C}.Release|Any CPU.Build.0 = Release|Any CPU
+<<<<<<< HEAD
```
Comment (mtran45) [inline src/MovieSeller.sln:22]: Remove merge conflict

File: src/MovieSellerDb/Program.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+using System.Reflection;
+
+namespace MovieSellerDb
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            IConfiguration Configuration = new ConfigurationBuilder()
+            .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
```
Comment (mtran45) [inline src/MovieSellerDb/Program.cs:14]: Indent these lines startin with .

File: src/MovieSellerDb/Scripts/001-CreateDatabase.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿CREATE TABLE [dbo].[user]
+(
+	[Id] INT NOT NULL PRIMARY KEY,
```
Comment (mtran45) [inline src/MovieSellerDb/Scripts/001-CreateDatabase.sql:3]:
```suggestion
	[Id] INT IDENTITY NOT NULL PRIMARY KEY, 
```

File: src/MovieSellerDb/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (mtran45) [inline src/MovieSellerDb/appsettings.json:1]: Hmm I want to make a comment on this file and not a specific line. It's possible in Bitbucket but not sure if you can do it in Github..

Anyway, my comment:
This should probably be a template file without anything in the connection strings. And then the dev database connection string will be specified in appsettings.Development.json

But we can sort that out later I suppose

File: src/MovieSellerDb/appsettings.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+
```
Comment (mtran45) [inline src/MovieSellerDb/appsettings.json:9]: Remove empty line

**Review by mtran45 — CHANGES_REQUESTED** [review]

File: src/MovieSellerDb/Scripts/001-CreateDatabase.sql (hunk 3 above)
Comment (kenash-dev) [reply src/MovieSellerDb/Scripts/001-CreateDatabase.sql:3]: done

File: src/MovieSellerDb/appsettings.json (hunk 5 above)
Comment (kenash-dev) [reply src/MovieSellerDb/appsettings.json:9]: Not an empty line, it's a new line, probably webviewer is showing more space, check in  your VS once before approving.

**Review by mtran45 — APPROVED** [review]

**Pull request merged**